dart/biomechanics/DynamicsFitter.hpp

```diff
@@ -1434,7 +1434,14 @@ class DynamicsFitter
       int maxTrialsToSolveMassOver = 4,
       bool detectExternalForce = true,
       int driftCorrectionBlurRadius = 250,
-      int driftCorrectionBlurInterval = 250);
+      int driftCorrectionBlurInterval = 250,
+      s_t regularizeUnobservedTimesteps = 0.01);
```

Comment:
The default value for `regularizeUnobservedTimesteps` was changed from 0.1 to 0.01 here -- will that have any meaningful effect on the dynamics fitter algorithm?